upgrade-cluster.yml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: all
+  any_errors_fatal: true
+  gather_facts: true
+
+- hosts: all:!network-storage
+  any_errors_fatal: true
+  roles:
+    - { role: kubernetes/preinstall, tags: preinstall }
+
+- hosts: etcd:!k8s-cluster
+  any_errors_fatal: true
+  serial: 1
+  roles:
+    - { role: etcd, tags: etcd }
+
+- hosts: kube-node
+  any_errors_fatal: true
+  serial: 1
```

Comment:
I strongly suggest we use serial: 20% so that this runs in 5 sections. One-at-a-time upgrade will take several hours for 50 nodes. We can put notes on this in the documentation.